mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -60,16 +60,16 @@ class LLVM_IntArithmeticOpWithOverflowFlag<string mnemonic, string instName,
     LLVM_ArithmeticOpBase<AnySignlessInteger, mnemonic, instName,
     !listconcat([DeclareOpInterfaceMethods<IntegerOverflowFlagsInterface>], traits)> {
   dag iofArg = (
-    ins DefaultValuedAttr<LLVM_IntegerOverflowFlagsAttr, "{}">:$overflowFlags);
```

Comment:
Is LLVM_IntegerOverflowFlagsAttr still used after this PR or could it be removed as well?

Thanks for the change!